bins/stackabled/src/cli.rs

```diff
@@ -16,4 +16,10 @@ pub struct Cli {
     /// Address the server binds to
     #[arg(short, long, default_value = "127.0.0.1", env = "STACKABLED_ADDRESS")]
     pub address: IpAddr,
+
+    /// Path to the password database, can be generated with the Apache `htpasswd` utility
+    ///
+    /// Only bcrypt passwords are supported (`htpasswd -B`).
+    #[arg(long, env = "STACKABLED_HTPASSWD")]
```

Comment:
`STACKABLED_HTPASSWD_PATH` or `STACKABLED_HTPASSWD_FILE` might be better here.

Reply:
15f60b4281a255b46dc1980a084793863e0781b8